File: HotelBookingApplication/src/main/java/com/cg/hbm/dto/HotelDTO.java
package com.cg.hbm.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class HotelDTO {
	
	private int hotelId;
	private String city;
	private String hotelName;
	private String address;
	private String imageName;
	private String description;
	private double avgRatePerDay;
	private String email;
	private String phone;
	private String website;
	private int starRating;
	
	
	
	

}

File: src/app/hotel-dto.ts
export class HotelDTO {
    hotelId:number;
    city:string;
    hotelName:string;
    address:String;
    description:string;
    avgRatePerDay:number;
    email:string;
    phone:string;
    starRating:number;
    website:string;
    imageName:string

    

    constructor(
        hotelId:number,
        city:string,
        hotelName:string,
        address:String,
        description:string,
        avgRatePerDay:number,
        email:string,
        phone:string,
        starRating:number,
        website:string,
        imageName:string
        
        
    )
    {
          this.hotelId=hotelId;
          this.city=city;
          this.hotelName=hotelName;
          this.address=address;
          this.description=description;
          this.avgRatePerDay=avgRatePerDay;
          this.email=email;
          this.phone=phone;
          this.starRating=starRating;
          this.website=website;
          this.imageName=imageName

    }

}//end of class